From b1f30dac48a53832f677e4f0050809a5debe8cd8 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Thu, 30 Oct 2014 16:14:47 +0100
Subject: [PATCH] 2

---
 init.rc | 29 ++++++++++++++++++++++-------
 1 file changed, 22 insertions(+), 7 deletions(-)

diff --git a/init.rc b/init.rc
index 5932bc9..94eceb5 100755
--- a/init.rc
+++ b/init.rc
@@ -59,6 +59,9 @@ on init
     mkdir /mnt/shell 0750 shell shell
     mkdir /mnt/media_rw 0700 media_rw media_rw
     mkdir /storage 0751 root sdcard_r
+
+    # Create mountpoint so Dalvik can mark as slave in zygotes.
+    mount tmpfs tmpfs /storage mode=0050,uid=0,gid=1028
         
     # Directory for putting things only root should see.
     mkdir /mnt/secure 0700 root root
@@ -135,6 +138,10 @@ on init
 # This is needed by any process that uses socket tagging.
     chmod 0644 /dev/xt_qtaguid
 
+# ko files for exfat -> AP: modules instead compiled into kernel
+    insmod /lib/modules/exfat_core.ko
+    insmod /lib/modules/exfat_fs.ko
+
 on fs
 # mount mtd partitions
     # Mount /system rw first to give the filesystem a chance to save a checkpoint
@@ -770,12 +777,12 @@ service powersnd /system/bin/samsungpowersoundplay
      disabled
      oneshot
 
-service bootanim /system/bin/bootanimation
-    class main
-    user graphics
-    group graphics system
-    disabled
-    oneshot
+#service bootanim /system/bin/bootanimation
+#    class main
+#    user graphics
+#    group graphics system
+#    disabled
+#    oneshot
 
 service installd /system/bin/installd
     class main
@@ -889,4 +896,12 @@ service pppoe /system/bin/pppoewrapper
     class main
     disabled
     oneshot
-# PPPOE SUPPORT
\ No newline at end of file
+# PPPOE SUPPORT
+# Call Boeffla-Kernel init script
+
+service boeffla-init /sbin/boeffla-init-starter.sh
+    class main
+    user root
+    group root
+    oneshot
+    
-- 
1.9.1

