From 3d14bd9f80628500bce74a91018342fb94ee31cb Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Sat, 20 Apr 2013 16:45:12 +0200
Subject: [PATCH] 2

---
 init.rc |   19 ++++++++++---------
 1 file changed, 10 insertions(+), 9 deletions(-)

diff --git a/init.rc b/init.rc
index 3a5c03a..e27457c 100755
--- a/init.rc
+++ b/init.rc
@@ -766,13 +766,6 @@ service bootanim /system/bin/bootanimation
     disabled
     oneshot

-service samsungani /system/bin/samsungani
-    class main
-    user graphics
-    group graphics system
-    disabled
-    oneshot
-
 service dbus /system/bin/dbus-daemon --system --nofork
     class main
     socket dbus stream 660 bluetooth bluetooth
@@ -886,3 +879,10 @@ on property:ro.dumpstate.dmesg=1
 on property:init.svc.media=restarting
     exec icd_check
     start icd
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class core
+    user root
+    oneshot
+
-- 
1.7.9.5

